ci-operator/step-registry/baremetal/lab/pre/firewall/baremetal-lab-pre-firewall-commands.sh

```diff
@@ -13,6 +13,25 @@ SSHOPTS=(-o 'ConnectTimeout=5'
   -o LogLevel=ERROR
   -i "${CLUSTER_PROFILE_DIR}/ssh-key")
 
+setup_proxy() {
+  proxy="$(<"${CLUSTER_PROFILE_DIR}/proxy")"
+  cat <<EOF > "${SHARED_DIR}/proxy-conf.sh"
+  export HTTP_PROXY=${proxy}
+  export HTTPS_PROXY=${proxy}
+  export NO_PROXY="localhost,127.0.0.1"
```

Comment:
Mmm, I think we missed something here, as we don't want to reach the proxy for the internal names.

```suggestion
  export NO_PROXY="localhost,127.0.0.1,$(<${CLUSTER_PROFILE_DIR}/base_domain)"
```